Mathlib/CategoryTheory/Monoidal/Opposite.lean

```diff
@@ -314,11 +314,38 @@ variable (C)
 /-- The (identity) equivalence between `Cᴹᵒᵖ` and `C`. -/
 @[simps!] def MonoidalOpposite.unmopEquiv : Cᴹᵒᵖ ≌ C := (mopEquiv C).symm
 
--- todo: upgrade to monoidal equivalence
 /-- The equivalence between `C` and its monoidal opposite's monoidal opposite. -/
 @[simps!] def MonoidalOpposite.mopMopEquivalence : Cᴹᵒᵖᴹᵒᵖ ≌ C :=
   .trans (MonoidalOpposite.unmopEquiv Cᴹᵒᵖ) (MonoidalOpposite.unmopEquiv C)
 
+@[simps!]
+instance : (MonoidalOpposite.mopMopEquivalence C).functor.Monoidal where
```

Comment:
So that the simp lemmas have reasonable names:
```suggestion
instance MonoidalOpposite.mopMopEquivalenceFunctorMonoidal :
  (MonoidalOpposite.mopMopEquivalence C).functor.Monoidal where
```